src/components/common/Form/ValidationMessage.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div class="validation-message">
+    <p>
+      <slot></slot>
+    </p>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.validation-message {
+  margin-top: 8px;
+  font-size: 14px;
+  font-family: $font-main;
+  color: $color-text-main;
+  letter-spacing: 0.04em;
+  line-height: 170%;
+
+  &[id='2'] {
+    margin-top: 0;
+  }
+
+  &[id='3'] {
+    margin-top: 0;
+  }
```

Comment:
When would the validation message even have such an ID (also it shouldn't base it's styles on it) 🤔

Reply:
I tried to address some style issues with that, but there is definitely a better way 😸